config/locales/client.en.yml

```diff
@@ -229,6 +229,9 @@ en:
       broken_theme_alert: "Your site may not work because a theme / component has errors."
       error_caused_by: "Caused by '%{name}'. <a target='blank' href='%{path}'>Click here</a> to update, reconfigure or disable."
       only_admins: "(this message is only shown to site administrators)"
+      horizon:
```

Comment:
I don't think we need to have this under a horizon group. We already have `topic_statuses.pinned`:

https://github.com/discourse/discourse/blob/3f666262cda2c269b11aa846d5cf7bdd453274ec/config/locales/client.en.yml#L4452-L4453

So we could just add Hot under here